src/hooks/usePickerInput.ts

```diff
@@ -37,12 +42,24 @@ export default function usePickerInput({
 
   const valueChangedRef = useRef<boolean>(false);
 
+  const preventDefaultRef = useRef<boolean>(false);
+
   const inputProps: React.DOMAttributes<HTMLInputElement> = {
     onMouseDown: () => {
       setTyping(true);
       triggerOpen(true);
     },
     onKeyDown: e => {
+      if (onKeyDown) {
```

Comment:
The `onKeyDown` should always have value?

Reply:
no, i will remove that line